release_bot/configuration.py

```diff
@@ -127,6 +130,14 @@ def load_release_conf(self, conf):
             msg = "Can't trigger on issue if 'github_username' is not known, disabling"
             self.logger.warning(msg)
             parsed_conf['trigger_on_issue'] = False
+        pypi_name = get_pypi_project_name_from_setup_cfg()
+        if pypi_name is None:
```

Comment:
Also, the order of where to get the pypi project name should be:
1. `release-conf.yaml` (i.e. `parsed_conf.get('pypi_project')`)
2. `setup.cfg` (i.e. `get_pypi_project_name_from_setup_cfg()`)
3. repository_name (i.e. `self.repository_name`)